pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -59,6 +59,15 @@ type CronWorkflowSpec struct {
 	Timezone string `json:"timezone,omitempty" protobuf:"bytes,8,opt,name=timezone"`
 	// WorkflowMetadata contains some metadata of the workflow to be run
 	WorkflowMetadata *metav1.ObjectMeta `json:"workflowMetadata,omitempty" protobuf:"bytes,9,opt,name=workflowMeta"`
+	// StopStrategy defines if the cron workflow will stop being triggered once a certain condition has been reached, involving a number of runs of the workflow
```

Comment:
Similar to https://github.com/argoproj/argo-workflows/pull/12616#discussion_r1495330587, we'd want to put a version specifier here since this pops up in the Field Reference.

```suggestion
	// v3.6 and after: StopStrategy defines if the CronWorkflow should stop scheduling based on a condition
```

- simpler and more direct, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
  - "involving a number of runs of the workflow" is described in the `condition` definition, so we can leave that out here (it is also a bit verbose)
- consistently use "CronWorkflow"
- I think "scheduling" as you used below in the `Condition` description is clearer wording than "triggering". "Triggering" is a bit ambiguous and not used in the docs, whereas "Schedule" is an existing term